src/foundations/shape/Shape.tsx

```diff
@@ -27,39 +34,78 @@ const getShapeDimensions = (props: PShape): string => {
     return width + height;
 };
 
-const getBorderDefinition = (props: PShape): string => {
-    const { borderColor = 'var(--default-border-color)', borderWidth } = props;
+const getBorderRadius = (radius: TShapeBorderRadius): string => {
+    if (SharedUtils.isString(radius)) {
+        return radius === 'circle' ? '50%' : '10000px';
+    }
+
+    return `${radius}px`;
+};
+
+const getElevationValue = (elevation: TShapeElevationLevel, opacity: number): string =>
+    `0 ${SHAPE_ELEVATION_DEFINITIONS[elevation].y}px ${SHAPE_ELEVATION_DEFINITIONS[elevation].blur}px 0 rgba(0,0,0,${opacity})`;
+
+const getElevation = ({
```

Comment:
I think we should be able to define a color shade for the elevation box-shadow. Sometimes they want it blue, sometimes neutral.

Reply:
This is something we need to clarify with Design as it's not in the design docs yet. If it's a legit use, they will need to update the docs.